### ISSUE TITLE
missing support for fill-lower and fill-upper

### ISSUE THREAD

**Issue by SlimMarten** [seaneking/postcss-input-style#2]
A bundled selector for (::-moz-range-progress, ::-ms-fill-lower) would be nice, as well as a bundled selector to style IEs range ticks. (::-ms-ticks-before, ::-ms-ticks-after, ::-ms-track)

**Comment by seaneking** [comment]
The problem with fill-lower and ticks is that they're vendor specific features, so using them here would only apply to those browsers, and at that point why not just use the vendor pseudos directly?

**Comment by SlimMarten** [comment]
well thats right, but imo it would be nicer to write ::fill-lower instead of both, ::-moz-range-progress and ::-ms-fill-lower. Same for the ticks. A shorthand is easier to remember and it could include all necessary vendor prefixes to cover all supporting browsers.

**Comment by seaneking** [comment]
I thought ticks were an ms only thing? 

Might consider including progress/fill-lower, with the caveat that WebKit and blink aren't supported. I'm flat out atm though, would accept a PR if you'd like to submit one?

**Comment by SlimMarten** [comment]
yes, they are ms only, but to style them properly you have to use three different prefixed selectors.
I am busy until the end of march, but I will let you know if I get the time to include the update.

**Comment by seaneking** [comment]
:+1:
